### ISSUE TITLE
There is no .changeset folder

### ISSUE THREAD

**Issue by tpluscode** [atlassian/changesets#545]
## Affected Packages

`pkg:cli`

## Problem

A typical configuration of a monorepo is that packages are two levels deep.

```
.
├── .changesets
└── packages/
    ├── package-a
    └── package-b
```

With the `changeset` command can be called in the root and in `./packages` but when the current working dir is deeper it errors with 

```
🦋  error There is no .changeset folder. 
```

## Proposed solution

I would expect the CLI to go up the directory tree until it find that `.changeset` folder.

**Comment by Andarist** [comment]
Detecting the root directory is notoriously problematic in a lot of tools. There is no well-defined indicator of what is the root and what's not. So I'm a little bit hesitant about this.

OTOH, for our case, it seems that we shouldn't get into much trouble by doing this. I'm wondering though - what's your use case? It's the first time somebody has asked for it.

**Comment by tpluscode** [comment]
Well, nothing really fancy. Just that I would have my terminal open and sometime `cd` into individual packages to call some scripts or whatever. Then if I don't go back and try `yarn changeset` it fails

I'm sure others had that but it's a minor nuisance at best

**Comment by Andarist** [comment]
@mitchellhamilton any objections to this?

**Comment by mitchellhamilton** [comment]
Yes, let's do this. For detecting the root, @manypkg/get-packages already does it, let's use that.

**Comment by kortin99** [comment]
Is this project dead? Why the issue from 2021 still in an open status?

**Comment by Andarist** [comment]
> Is this project dead?

No. We don't have a lot of time to work on it though. 

> Why the issue from 2021 still in an open status?

Perhaps because nobody provided a fix for this. This is an open-source project - I encourage you to fix this and put up a PR for review.

**Comment by kortin99** [comment]
> Perhaps because nobody provided a fix for this. This is an open-source project - I encourage you to fix this and put up a PR for review.

I've noticed that @chaance submitted a PR about a year ago, but it seems there hasn't been any follow-up since then.

**Comment by luisadame** [comment]
I've opened this PR: https://github.com/changesets/changesets/pull/1806 fixing this issue using one of the packages mentioned in this previous comment here: https://github.com/changesets/changesets/issues/545#issuecomment-804423726.

Apart from having its own tests I've tested in a separate project that uses this tool and it seems to be working as expected.

Let me know what you think and opened to any suggestion, thanks!
